### PR TITLE
Add connect_to_ec2_instance.md

### PR DESCRIPTION
**Pull request by dilyand**

### CODE REVIEW

File: connect_to_ec2_instance.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+Now you can log into the Bastion box:
+
+```bash
+ssh -A -i ~/path/to/my_key ec2-user@{BASTION_PUBLIC_IP}
+```
+
+Once you're in the box, you can connect to any running EC2 instance using its private IP address:
+
+```bash
+ssh ec2-user@{EC2_INSTANCE_IP}
```
Comment (oguzhanunlu) [inline connect_to_ec2_instance.md:39]: iirc, username should be `ubuntu`, not `ec2-user`. Have you tried `ec2-user` and it worked fine?

File: connect_to_ec2_instance.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+```
+
+## Setting up port forwarding between and EC2 instance and localhost via Bastion
+
+This is useful if you want to run some profiling app on the EC2 machine.
+
+First, activate port forwarding between your local machine and Bastion:
+
+```bash
+ssh -A -X -i ~/path/to/my_key ec2-user@{BASTION_PUBLIC_IP} -4NL 5555:localhost:5555
```
Comment (oguzhanunlu) [inline connect_to_ec2_instance.md:49]: I think it is worth mentioning which machine we execute this command at even though it is implicitly clear.

**Comment by benjben** [comment]
Thanks for creating the page @dilyand ! I will add some comments, but first, I think that this should be internal resources, this is dealing with how we can ssh machines.

File: connect_to_ec2_instance.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+```
+
+This will open a connection that will stay open, so you'll need a new shell for the next command.
+
+Next up, set up port forwarding between the Bastion host and the EC2 box:
+
+```bash
+ssh ubuntu@{EC2_INSTANCE_IP} -4NL 5555:localhost:5555
```
Comment (oguzhanunlu) [inline connect_to_ec2_instance.md:57]: Same comment regarding which machine to execute at

**Pull request closed without merging**